src/widgets/DataTable/DataTableRow.js

```diff
@@ -87,8 +87,8 @@ const DataTableRow = React.memo(
           const cellAlignment = alignText || 'left';
 
           switch (type) {
-            case COLUMN_TYPES.STRING_EDITABLE:
-            case COLUMN_TYPES.NUMERIC_EDITABLE:
+            case COLUMN_TYPES.EDITABLE_STRING:
+            case COLUMN_TYPES.EDITABLE_NUMERIC:
               return (
                 <TextInputCell
                   key={columnKey}
```

Comment:
line 106 will need updating:

`type === COLUMN_TYPES.NUMERIC ? COLUMN_TYPES.NUMERIC_EDITABLE : 'default'`

should be 

`type === COLUMN_TYPES.NUMERIC ? COLUMN_TYPES.EDITABLE_NUMERIC : 'default'`

Reply:
fc2ec4a508c62d362e524ebcd5794ca99f79f40f thanks!